src/components/editor/MonacoEditor.tsx

```diff
@@ -187,10 +190,9 @@ function MonacoEditor({
                         theme={
                             theme.palette.mode === 'light' ? 'vs' : 'vs-dark'
                         }
-                        defaultValue={intl.formatMessage({
-                            id: 'common.loading',
-                        })}
```

Comment:
Setting the default value to loading was a bad idea and caused a weird UX. A user could `ctrl+z` back to the loading value.